sycl/CMakeLists.txt

```diff
@@ -210,6 +210,7 @@ configure_file("${version_header}.in" "${version_header}")
 # Copy SYCL headers
 add_custom_target(sycl-headers ALL
 COMMAND ${CMAKE_COMMAND} -E copy_directory ${sycl_inc_dir}/CL ${SYCL_INCLUDE_BUILD_DIR}/CL
+COMMAND ${CMAKE_COMMAND} -E copy_directory ${sycl_inc_dir}/sycl ${SYCL_INCLUDE_BUILD_DIR}/sycl
```

Comment:
isn't it better to copy ${sycl_inc_dir} to ${SYCL_INCLUDE_BUILD_DIR}? That will allow to avoid multiple command per subfolder.

Reply:
Done.